src/main/java/com/google/error_analyzer/data/constant/LogFields.java

```diff
@@ -17,10 +17,10 @@
     public static final String LOG_TEXT = "logText";
     public static final String LOG_LINE_NUMBER = "logLineNumber";
     public static final String LOG = "Log";
-    public static final String INDEX = "index";
```

Comment:
isnt it used anywhere else?

Reply:
nope i replaced it with fileName so not needed now